src/open_inwoner/cms/cases/tests/test_contactform.py

```diff
@@ -442,6 +443,9 @@ def test_form_success_with_api_eherkenning_user(self, m):
             with self.subTest(
                 use_rsin_for_innNnpId_query_parameter=use_rsin_for_innNnpId_query_parameter
             ):
+                # FIXME for some reason creating the user outside of the loop
+                # makes the second iteration fail?
+                User.objects.filter(kvk="12345678").delete()
```

Comment:
Same/similar issue as in https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/accounts/tests/test_profile_views.py#L547, where the mock has to be recreated because it is overriden somehow? Something's up with our mock machinery that needs to be fixed, though not sure if this can be done as part of your ticket; perhaps create a new one for this.

Reply:
Agree these delete()'s are a bit smelly.